src/Disks/ObjectStorages/StoredObject.h

```diff
@@ -21,6 +21,13 @@ struct StoredObject
 
     StoredObject() = default;
 
+    StoredObject(
```

Comment:
it seems so bad to have 4 constructors of `StoredObject`, even before 3 is not good (the last time I saw this file there was only 1), may be delete all and make one with default values for `bytes_size` and `local_path`?